README.md

```diff
@@ -1,5 +1,41 @@
-# Upbound Database-as-a-Service Configuration
 
-Multicloud Upbound DBaaS.
 
-One Claim to rule them all. See [examples](./examples).
+# database Configuration
```

Comment:
```suggestion
# Database-as-a-service Configuration
```
Standalone Database configurations can be found at https://github.com/upbound/configuration-aws-database , https://github.com/upbound/configuration-azure-database and https://github.com/upbound/configuration-gcp-database

This one is combining them to provide multicloud capabilities through the single abstraction.